sdks/java/container/common.gradle

```diff
@@ -45,6 +45,8 @@ dependencies {
 }
 
 task copyDockerfileDependencies(type: Copy) {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE 
```

Comment:
I think this might be a new things we need for Gradle 7 (https://github.com/gradle/gradle/issues/17236)

Reply:
The comments on that bug are 😨